tools/unicodeCat.js

```diff
@@ -3,10 +3,9 @@
 const request = require('request');
```

Comment:
In eb648f4b06dc2ce84e028ba139878439e2933322 I added a comment to explain the code here, both purpose and algorithm.